lib/hijack/wrap_ddp_stringify.js

```diff
@@ -0,0 +1,33 @@
+wrapStringifyDDP = function() {
+  var originalStringifyDDP = DDPCommon.stringifyDDP;
+
+  DDPCommon.stringifyDDP = function(msg) {
+    var msgString = originalStringifyDDP(msg);
+    var msgSize = msgString.length;
+
+    var kadiraInfo = Kadira._getInfo(null, true);
+
+    if(kadiraInfo) {
+      if(kadiraInfo.trace.type === 'method') {
+        Kadira.models.methods.trackMsgSize(kadiraInfo.trace.name, msgSize);
+      } else if(kadiraInfo.trace.type === 'sub') {
+        Kadira.models.pubsub.trackMsgSize(kadiraInfo.trace.name, "initialSent", msgSize);
+      }
+
+      return msgString;
+    }
+
+    if(Kadira.env.currentSub) {
```

Comment:
currentSub is already set here: https://github.com/meteorhacks/kadira/blob/59b1b27035ad4a2ad5a911fad37ec8eb8b7aed40/lib/hijack/wrap_subscription.js#L85